.changelog/4317.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Removed support for IN124 - 'is param valid is hidden' validation from old-validate flow and prevent the validation from being ignored.
```

Comment:
add a note that this functionality can be found in the new validate and how to activate